source/Application/Controller/Admin/ShopLicense.php

```diff
@@ -117,8 +117,11 @@ protected function _fetchCurVersionInfo($sUrl)
         $oCurl->setMethod("POST");
         $oCurl->setUrl($sUrl . "/" . $sLang);
         $oCurl->setParameters($aParams);
-        $sOutput = $oCurl->execute();
-
+        try {
+            $sOutput = $oCurl->execute();
+        } catch (oxException $ex) {
```

Comment:
no sadly it throws an basic "oxException" on curl errors. See
https://github.com/OXID-eSales/oxideshop_ce/blob/master/source/Core/Curl.php#L314

I guess it would be possible to create a "CurlException" in this place without any BC break. I would like to see that happen but i will not include it in this pull request to keep things atomic.